examples/hello-world/hello-flower/README.md

```diff
@@ -14,15 +14,24 @@ $ tree jobs/hello-flwr-pt
 ```
 Note, this code is directly copied from Flower's [app-pytorch](https://github.com/adap/flower/tree/main/examples/app-pytorch) example.
 
-## Install dependencies
+## 1. Install dependencies
 To run this job with NVFlare, we first need to install the dependencies.
 ```bash
 pip install -r requirements.txt
 ```
 
-## Run a simulation
+## 2.1 Run a simulation
 
 Next, we run 2 Flower clients and Flower Server in parallel using NVFlare's simulator.
 ```bash
 nvflare simulator jobs/hello-flwr-pt -n 2 -t 2 -w /tmp/nvflare/flwr
 ```
+
+## 2.2 Run a simulation with TensorBoard streaming
+
+Next, we run 2 Flower clients and Flower Server in parallel using NVFlare while streaming 
+the TensorBoard metrics to the server at each iteration using NVFlare's metric streaming.
+
+```bash
+CLIENT_API_TYPE="EX_PROCESS_API" nvflare simulator jobs/hello-flwr-pt_tb_streaming -n 2 -t 2 -w /tmp/nvflare/flwr_tb_streaming
```

Comment:
Why do we need to set `CLIENT_API_TYPE` here? Couldn't there be a default option that is used if the environment variable is not specified?

Reply:
If we keep the env variable here, we probably should add some explanation as to why this is needed. Is there documentation? cc @nvkevlu

Reply:
Since flower process is launched outside, the default IN_PROCESS_API will not work